pkg/scalers/kafka_scaler.go

```diff
@@ -49,6 +50,7 @@ const (
 	kafkaAuthModeForSaslScramSha256 kafkaAuthMode = "sasl_scram_sha256"
 	kafkaAuthModeForSaslScramSha512 kafkaAuthMode = "sasl_scram_sha512"
 	kafkaAuthModeForSaslSSL         kafkaAuthMode = "sasl_ssl"
+	kafkaAuthModeForSaslSSLPlain    kafkaAuthMode = "sasl_ssl_plain"
```

Comment:
what's the difference between `sasl_ssl_plain` and `sasl_plaintext`, does it still use ssl but without a client cert?

Reply:
That is correct, SASL_PLAINTEXT does not use SSL and the current SASL_SSL is configured to ask for certs.
Event Hubs authenticates with SASL_SSL with creds passed as PLAIN.